de.bforartists.Bforartists.json

```diff
@@ -0,0 +1,170 @@
+{
+    "id": "de.bforartists.Bforartists",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "22.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "bforartists",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=host",
+        "--filesystem=/run/spnav.sock:ro",
+        "--env=SPNAV_SOCKET=/run/spnav.sock",
+        "--env=TMP_DIR=/tmp",
+        "--env=TMP=/tmp"
+    ],
+    "add-extensions": {
+        "org.bforartists.bforartists.Codecs": {
+            "directory": "lib/codecs",
+            "add-ld-path": "lib",
+            "bundle": true,
+            "autodelete": true
+        },
+        "org.freedesktop.Platform.ffmpeg-full": {
+            "directory": "lib/ffmpeg",
+            "version": "22.08",
+            "add-ld-path": "."
+        }
+    },
+    "cleanup-commands": [
+        "mkdir -p /app/lib/ffmpeg"
```

Comment:
cleanup is to cleanup, not to create.

Please add this to the post-install of the main module.